lib/datasets/california-housing.rb

```diff
@@ -0,0 +1,64 @@
+require "csv"
+
+require_relative "dataset"
+require "rubygems/package"
+require "zlib"
+
+module Datasets
+  class CaliforniaHousing < Dataset
+    Record = Struct.new(:longitude,
+                        :latitude,
+                        :housingMedianAge,
+                        :totalRooms,
+                        :totalBedrooms,
+                        :population,
+                        :households,
+                        :medianIncome,
+                        :medianHouseValue)
+
+    def initialize
+      super()
+      @metadata.id = "california-housing"
+      @metadata.name = "California Housing"
+      @metadata.url = "https://www.dcc.fc.up.pt/~ltorgo/Regression/cal_housing.html"
+      @metadata.licenses = ["Unknown"]
+      @metadata.description =  <<~DESCRIPTION
+Housing information from the 1990 census used in
+Pace, R. Kelley and Ronald Barry,
+"Sparse Spatial Autoregressions",
+Statistics and Probability Letters, 33 (1997) 291-297.
```

Comment:
Could you mention something about StatLib because it says:

http://lib.stat.cmu.edu/datasets/

> ##  Credit where credit is due
>
>  If you use an algorithm, dataset, or other information from StatLib, please acknowledge both StatLib and the original contributor of the material.

Reply:
Sure.